python_backend/api/conversion_routes.py

```
@@ -0,0 +1,140 @@
+        if not result.get("success"):
+            raise HTTPException(status_code=500, detail=result.get("error", "피드백 처리 중 오류가 발생했습니다."))
+        
+        return result
+        
+    except HTTPException:
+        raise
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"서버 오류: {str(e)}")
+
+@router.get("/health")
+@app.get("/health", tags=["Health Check"], summary="서버 상태 확인")
```

Ruff (0.12.2)

Undefined name `app`

(F821)